token-locker/src/lib.rs

```diff
@@ -215,37 +222,30 @@ impl Contract {
         assert!(verification_success, "Failed to verify the proof");
         let required_deposit = self.record_proof(&proof);
 
-        assert!(
-            env::attached_deposit()
-                >= required_deposit
-        );
+        assert!(env::attached_deposit() >= required_deposit);
 
         let Recipient { target, message } = parse_recipient(new_owner_id.into());
 
-        env::log_str(format!("Finish deposit. Token:{} Target:{} Message:{:?}", token, target, message).as_str());
+        env::log_str(
+            format!(
+                "Finish deposit. Token:{} Target:{} Message:{:?}",
+                token, target, message
+            )
+            .as_str(),
+        );
 
         match message {
             Some(message) => ext_token::ext(token.try_into().unwrap())
-            .with_attached_deposit(1)
-            .with_static_gas(FT_TRANSFER_CALL_GAS)
-            .ft_transfer_call(
-                    target.try_into().unwrap(),
-                    amount.into(),
-                    None,
-                    message,
-                ),
+                .with_attached_deposit(1)
```

Comment:
Let's use these constants now
```suggestion
                .with_attached_deposit(near_sdk::ONE_YOCTO)
```